quodlibet/ext/query/pythonexpression.py

```diff
@@ -18,16 +18,13 @@ class PythonQuery(QueryPlugin):
     PLUGIN_NAME = _("Python Query")
     PLUGIN_DESC = _("🐍Use Python expressions in queries.")
     key = 'python'
-    usage = """
-<b><tt>@(python: expression)</tt></b>
-
-The variable <tt>s</tt> (or <tt>a</tt>) is the song / album being matched.
+    usage = "<b><tt>" + _("@(python: expression)") + "</tt></b>\n\n" + _(
```

Comment:
Let's split this up into varaibles 

e.g. 
```python
query_syntax = _("@(python: expression)")
```
etc

Reply:
Have done that for all the query plugins now.